### PR TITLE
feature/AT-RSC-UI-Task Added Admin Task Views and Sections

### PR DESCRIPTION
**Pull request by LuisAPerez224**
feature/AT-RSC-UI-Task Added Admin Task Views and Sections

### CODE REVIEW

**Review by irvingesp2190 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -33,8 +33,8 @@
     "devDependencies": {
         "dotenv": "^8.2.0",
         "handlebars": "^4.7.6",
-        "nodemon": "^2.0.4",
-        "npm-check-updates": "^9.0.4"
+        "nodemon": "^2.0.5",
```
Comment (at-internship) [inline package.json]: why this update?

File: src/controllers/at-resources.controller.js [hunk 2]
```diff
@@ -1,9 +1,37 @@
...
-rscCtrl.dashboard = async(req, res) => {
+rscCtrl.renderDashboard = async(req, res) => {
     console.log("--> rscCtrl.dashboard");
     res.render("dashboard");
 };
+//---------------------------------------
+
+
+
+// AT-SSO - Admin - Dashboard - Render Add User Form
```
Comment (at-internship) [inline src/controllers/at-resources.controller.js]: Should be  `// AT-RESOURCES - Task - Render Add Task Form`

File: src/routes/at-resources.routes.js [hunk 3]
```diff
@@ -3,17 +3,28 @@ const router = express.Router();
...
+        updateTask } = require("../controllers/at-resources.controller");
 
 // Helpers
 // const { isAdmin } = require("../helpers/auth");
 
 // ============= Sub Routes =============
 
-// AT-RESOURCES - Index
-router.get('/', dashboard);
+router.get('/', renderDashboard);
```
Comment (at-internship) [inline src/routes/at-resources.routes.js:24]: Please add code comments for every route like `// AT-RESOURCES - Dashboard`

File: src/routes/at-resources.routes.js [hunk 4]
```diff
@@ -3,17 +3,28 @@ const router = express.Router();
...
 
-// AT-RESOURCES - Dashboard
-router.get("/dashboard", dashboard);
+router.get("/dashboard", renderDashboard);
+
+router.get("/task/add", renderAddTaskForm);
```
Comment (at-internship) [inline src/routes/at-resources.routes.js:30]: `// AT-RESOURCES - Task - Add Task Form`

File: src/routes/at-resources.routes.js [hunk 5]
```diff
@@ -3,17 +3,28 @@ const router = express.Router();
...
+
+router.post("/task/add", addTask);
```
Comment (at-internship) [inline src/routes/at-resources.routes.js:33]: `// AT-RESOURCES - Task - Add`

File: src/routes/at-resources.routes.js [hunk 6]
```diff
@@ -3,17 +3,28 @@ const router = express.Router();
...
+
+router.get("/task/edit/:id", renderEditTaskForm);
```
Comment (at-internship) [inline src/routes/at-resources.routes.js:36]: `// AT-RESOURCES - Render Edit Task Form`

File: src/views/partials/menu-at-resources.hbs [hunk 7]
```diff
@@ -0,0 +1,132 @@
+<!-- MENU -->
+<ul class="nav flex-column">
+    <li class="nav-item">
+        <a class="nav-link" href="/at-resources/dashboard">
```
Comment (at-internship) [inline src/views/partials/menu-at-resources.hbs]: shoulde be `<a class="nav-link" href="/dashboard">`

File: src/views/partials/menu-at-resources.hbs [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+                stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"
+                class="feather feather-home">
+                <path d="M3 9l9-7 9 7v11a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2z"></path>
+                <polyline points="9 22 9 12 15 12 15 22"></polyline>
+            </svg>
+            Dashboard 
+        </a>
+    </li>
+    <li class="nav-item">
+        <a class="nav-link" href="/at-resources/backlog">
```
Comment (at-internship) [inline src/views/partials/menu-at-resources.hbs]: shoulde be `<a class="nav-link" href="/backlog">`

File: src/views/partials/menu-at-resources.hbs [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+                xmlns="http://www.w3.org/2000/svg">
+                <path
+                    d="M1 3.5a.5.5 0 0 1 .5-.5h13a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-13a.5.5 0 0 1-.5-.5v-1zm0 4a.5.5 0 0 1 .5-.5h5a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-5a.5.5 0 0 1-.5-.5v-1zm0 4a.5.5 0 0 1 .5-.5h1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-1a.5.5 0 0 1-.5-.5v-1zm8 0a.5.5 0 0 1 .5-.5h1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-1a.5.5 0 0 1-.5-.5v-1zm-4 0a.5.5 0 0 1 .5-.5h1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-1a.5.5 0 0 1-.5-.5v-1zm8 0a.5.5 0 0 1 .5-.5h1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-1a.5.5 0 0 1-.5-.5v-1zm-4-4a.5.5 0 0 1 .5-.5h5a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-5a.5.5 0 0 1-.5-.5v-1z">
+                </path>
+            </svg>
+            Backlog
+        </a>
+    </li>
+    <li class="nav-item">
+        <a class="nav-link" href="/at-resources/activeSprint">
```
Comment (at-internship) [inline src/views/partials/menu-at-resources.hbs]: shoulde be `<a class="nav-link" href="/sprint">`

File: src/views/partials/menu-at-resources.hbs [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+                <rect width="8" height="2" x="4" y="5" rx=".5"></rect>
+                <path
+                    d="M6 9.5a.5.5 0 0 1 .5-.5h6a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-6a.5.5 0 0 1-.5-.5v-1zm2 4a.5.5 0 0 1 .5-.5h7a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-7a.5.5 0 0 1-.5-.5v-1z">
+                </path>
+            </svg>
+            Active Sprints
+        </a>
+    </li>
+    <li class="nav-item">
+        <a class="nav-link" href="#">
```
Comment (at-internship) [inline src/views/partials/menu-at-resources.hbs]: shoulde be `<a class="nav-link" href="/members">`

File: src/views/partials/menu-at-resources.hbs [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+                xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd"
+                    d="M3 14s-1 0-1-1 1-4 6-4 6 3 6 4-1 1-1 1H3zm5-6a3 3 0 1 0 0-6 3 3 0 0 0 0 6z">
+                </path>
+            </svg>
+            Users / Members
+        </a>
+    </li>
+    <li class="nav-item">
+        <a class="nav-link" href="#">
```
Comment (at-internship) [inline src/views/partials/menu-at-resources.hbs]: shoulde be `<a class="nav-link" href="/teams">`

File: src/views/partials/menu-at-resources.hbs [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+                xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd"
+                    d="M7 14s-1 0-1-1 1-4 5-4 5 3 5 4-1 1-1 1H7zm4-6a3 3 0 1 0 0-6 3 3 0 0 0 0 6zm-5.784 6A2.238 2.238 0 0 1 5 13c0-1.355.68-2.75 1.936-3.72A6.325 6.325 0 0 0 5 9c-4 0-5 3-5 4s1 1 1 1h4.216zM4.5 8a2.5 2.5 0 1 0 0-5 2.5 2.5 0 0 0 0 5z">
+                </path>
+            </svg>
+            Teams
+        </a>
+    </li>
+    <li class="nav-item">
+        <a class="nav-link" href="#">
```
Comment (at-internship) [inline src/views/partials/menu-at-resources.hbs]: shoulde be `<a class="nav-link" href="/mentors">`

File: src/views/partials/menu-at-resources.hbs [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+                xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd"
+                    d="M8 .5c-.662 0-1.77.249-2.813.525a61.11 61.11 0 0 0-2.772.815 1.454 1.454 0 0 0-1.003 1.184c-.573 4.197.756 7.307 2.368 9.365a11.192 11.192 0 0 0 2.417 2.3c.371.256.715.451 1.007.586.27.124.558.225.796.225s.527-.101.796-.225c.292-.135.636-.33 1.007-.586a11.191 11.191 0 0 0 2.418-2.3c1.611-2.058 2.94-5.168 2.367-9.365a1.454 1.454 0 0 0-1.003-1.184 61.09 61.09 0 0 0-2.772-.815C9.77.749 8.663.5 8 .5zm2.854 6.354a.5.5 0 0 0-.708-.708L7.5 8.793 6.354 7.646a.5.5 0 1 0-.708.708l1.5 1.5a.5.5 0 0 0 .708 0l3-3z">
+                </path>
+            </svg>
+            Mentors
+        </a>
+    </li>
+    <li class="nav-item">
+        <a class="nav-link" href="#">
```
Comment (at-internship) [inline src/views/partials/menu-at-resources.hbs]: shoulde be `<a class="nav-link" href="/metrics">`

File: src/views/partials/menu-at-resources.hbs [hunk 14]
```diff
@@ -0,0 +1,132 @@
...
+                <rect width="4" height="9" x="6" y="6" rx="1"></rect>
+                <rect width="4" height="14" x="11" y="1" rx="1"></rect>
+            </svg>
+            Metrics / Reports
+        </a>
+    </li>
+</ul>
+<!-- MENU -->
+
+<!-- REPORTS -->
```
Comment (at-internship) [inline src/views/partials/menu-at-resources.hbs]: Remove this section

**Review by at-internship — CHANGES_REQUESTED** [review]

**Comment by at-internship** [comment]
This pull must be merged into develop not feature/AT-RSC-UI_Migration

**Review by at-internship — APPROVED** [review]
Approved

**Pull request merged**